File: JAVA/Petshop.java
public class Petshop {
    
    // atribut yang akan dipakai dan aksesnya di private kan
    private int id;
    private String nama_produk;
    private String kategori_produk;
    private double harga_produk;

    /* Constructor */

    public Petshop()
    {
        this.id = 0;
        this.nama_produk = "";
        this.kategori_produk = "";
        this.harga_produk = 0;
    }

    // Constructor dengan parameter.
    public Petshop(int id, String nama_produk, String kategori_produk, double harga_produk) {
        this.id = id;
        this.nama_produk = nama_produk;
        this.kategori_produk = kategori_produk;
        this.harga_produk = harga_produk;
    }

    /* Getter and Setter (ACCESSOR) */

    // Get id
    public int get_id() {
        return this.id;
    }

    // Set id
    public void set_id(int id) {
        this.id = id;
    }

    // Get nama produk
    public String get_nama_produk() {
        return this.nama_produk;
    }

    // Set nama produk
    public void set_nama_produk(String nama_produk) {
        this.nama_produk = nama_produk;
    }

    // Get kategori produk
    public String get_kategori_produk() {
        return this.kategori_produk;
    }

    // Set kategori produk
    public void set_kategori_produk(String kategori_produk) {
        this.kategori_produk = kategori_produk;
    }

    // Get harga produk
    public double get_harga_produk() {
        return this.harga_produk;
    }

    // Set harga produk
    public void set_harga_produk(double harga_produk) {
        this.harga_produk = harga_produk;
    }

    // Nahh, Java doesn't need a destructor
}

File: PYTHON/Petshop.py
class Petshop:

    # ATRIBUT-ATRIBUT YANG MAU DIINISIALISASIKAN
    __id = 0
    __namaProduk = ""
    __kategoriProduk = ""
    __hargaProduk = 0.0

    # CONSTRUCTOR DENGAN PARAMETER UNTUK ME-NEW KAN
    def __init__(self, id = 0, namaProduk = "", kategoriProduk = "", hargaProduk = 0.0):
        self.__id = id
        self.__namaProduk = namaProduk
        self.__kategoriProduk = kategoriProduk
        self.__hargaProduk = hargaProduk

    # Getter and Setter

    # get id
    def get_id(self): return self.__id

    # set id
    def set_id(self, id): self.__id = id

    # get nama produk
    def get_namaProduk(self): return self.__namaProduk

    # set nama produk
    def set_namaProduk(self, namaProduk): self.__namaProduk = namaProduk

    # get kategori produk
    def get_kategoriProduk(self): return self.__kategoriProduk

    # set kategori produk
    def set_kategoriProduk(self, kategoriProduk): self.__kategoriProduk = kategoriProduk

    # get harga produk
    def get_hargaProduk(self): return self.__hargaProduk

    # set harga produk
    def set_hargaProduk(self, hargaProduk): self.__hargaProduk = hargaProduk